MAUI/Polar-Charts/Axis/Axislabels.md

```diff
@@ -11,6 +11,44 @@ documentation: ug
 
 Axis labels are used to display the units, measures, or category values of an axis in a user-friendly way. They are generated based on the range and the values bound to the [XBindingPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html#Syncfusion_Maui_Charts_ChartSeries_XBindingPath) or [YBindingPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.XYDataSeries.html#Syncfusion_Maui_Charts_XYDataSeries_YBindingPath) properties of the series.
 
+## Positioning the labels
+
+The [LabelsPosition]() property is used to position the axis labels inside or outside the chart area. By default, [LabelsPosition]() is positioned `AxisElementPosition.Outside` the chart area.
+
+N> Polar chart supports LabelsPosition for secondary axis only.
```

Comment:
This is only applicable to the secondary axis of the polar chart.

Reply:
Updated the content